xml/net/server/protocol.xml

```diff
@@ -366,6 +422,7 @@
     </struct>
 
     <struct name="CharacterStats1">
+        <comment>Character stats data</comment>
```

Comment:
These could probably use better names and where they are used (CharacterStats1-4)